cmd/manager/main.go

```diff
@@ -48,6 +52,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Become the leader before proceeding
+	err = leader.Become(context.TODO(), "anthill-lock")
```

Comment:
`anthill-lock` can be a constant.

Reply:
It could, but I'd like to leave it since it's verbatim from the latest SDK scaffolding. Changing standard template stuff makes upgrading the sdk version considerably harder.